php/Terminus/Models/Environment.php

```diff
@@ -110,6 +110,27 @@ public function backupUrl($bucket, $element) {
   }
 
   /**
+   * Changes connection mode
+   *
+   * @param [string] $value Connection mode, "git" or "sftp"
+   * @return [Workflow] $workflow
+   */
+  public function changeConnectionMode($value) {
+    switch ($value) {
+      case 'git':
+        $workflow_name = 'enable_git_mode';
+        break;
+      case 'sftp':
+        $workflow_name = 'enable_on_server_development';
+        break;
+    }
+
+    $params   = array('environment' => $this->get('id'));
```

Comment:
True, but the check query is almost instantaneous and the workflow takes 20+ seconds so the efficiency isn't a huge issue. I'm happy leaving this as is though. There are no negative effects other than a little wait time and if that's critical to a script then the script can check directly.